__tests__/app.test.js

```diff
@@ -0,0 +1,49 @@
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const testData = require("../db/data/test-data/index");
+const request = require("supertest");
+const app = require("../app");
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+// ## Relevant HTTP Status Codes
+
+// - 200 OK
+// - 201 Created
+// - 204 No Content
+// - 400 Bad Request
+// - 404 Not Found
+// - 405 Method Not Allowed
+// - 418 I'm a teapot
+// - 422 Unprocessable Entity
+// - 500 Internal Server Error
```

Comment:
Nice list, althought we won't be needing 405, 418, 422

Reply:
done